infra_tooling_account/infra_tooling_account/infra_tooling_common_stack.py

```diff
@@ -19,12 +19,14 @@
 
 from lib.core.constants import CDKDeployExclusions, TimestreamRetention
 from lib.aws.aws_naming import AWSNaming
+from lib.settings.settings import Settings
 
 
 class InfraToolingCommonStack(Stack):
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         self.stage_name = kwargs.pop("stage_name", None)
         self.project_name = kwargs.pop("project_name", None)
+        self.settings: Settings = kwargs.pop("settings", None)
```

Comment:
Do we need settings component in common stack? Maybe it's better to pass it only when needed?

Reply:
Please disregard - viewed the PR till the end 🙃

Reply:
Actually, I did that on purpose - in case we'd need settings in common stack sooner or later and, more importantly, for unification (as all stacks' initialization works in the same way).

Just as a thought - we might even consider moving stage, project, settings init into SalmonStack parent class and, then, inherit from SalmonStack, not just CDK Stack